src/styles/components/Input.scss

```diff
@@ -0,0 +1,31 @@
+.Input {
+  &__container {
+    @extend .courier;
+    font-size: 1.125rem;
+    border-radius: 0.3rem;
+    border-width: 2px;
+    height: 4rem;
+    max-width: 60%;
+
+    @include media('lg-up') {
+      max-width: 100%;
+    }
+  }
+
+  &__container--full-width {
+    max-width: 100%;
```

Comment:
do you need this if you have width:100%?

Reply:
```.Input__container`` is base input style and ``max-width``` is set to ```60%```. So if I don't put any ```max-width``` in ```.Input__container--full-width``` it will set to ```60%```. Also if I don't set ```width: 100%``` input won't be fit to container. :)